product_docs/docs/pgd/3.7/bdr/release-notes.mdx

```diff
@@ -5,6 +5,53 @@ originalFilePath: release-notes.md
 
 ---
 
+## BDR 3.7.22 (2023 Aug 29)
+
+This is a maintenance release for BDR 3.7 that includes minor
+improvements as well as fixes for issues identified in previous
+versions.
+
+Also check the release notes for [pglogical 3.7.22](/pgd/3.7/pglogical/release-notes/#pglogical-3722) for resolved issues
+that affect BDR as well.
+
+
+### Resolved issues
+
+- Changed `bdr.autopartition_drop_partition()` signature to use text.
+
+- Autopartition: Drop partition iff it exists
+  It will help in recover from the cases when duplicate drop_partition workitems are created.
+
+- Fixed memory leak in `bdr.sequence_alloc` by modifying the missing catalog signature.
+
+- Prevented superuser check when GUC was specified on PG commandline.
+
+- Added check for malformed connection string.
```

Comment:
I think we should mention RT95453 and also explain that `create_node()` will now check and fail if connection string is malformed. This could be a user visible change

Reply:
Done :+1: